tests/backup/backup_partial_test.go

```diff
@@ -471,3 +474,323 @@ var _ = Describe("{PartialBackupSuccessWithPxVolumes}", func() {
 
 	})
 })
+
+var _ = Describe("{DifferentBackupStatusValidation}", Label(TestCaseLabelsMap[DifferentBackupStatusValidation]...), func() {
+	var (
+		numDeployments              = 2
+		srcClusterUid               string
+		destClusterUid              string
+		appNamespaces               []string
+		schedulePolicyName          string
+		schedulePolicyUid           string
+		scheduledBackups            []string
+		scheduledBackupCount        = 4
+		labelSelectors              = make(map[string]string)
+		scheduledAppContexts        = make([]*scheduler.Context, 0)
+		timeBetweenScheduledBackups = 10 * time.Second
+		cloudAccountName            string
+		cloudAccountUid             string
+		backupLocationName          string
+		backupLocationUid           string
+		backupLocationMap           map[string]string
+	)
+
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("DifferentBackupStatusValidation", "Verifies all the different backup status", nil, 299234, Sabrarhussaini, Q1FY25)
```

Comment:
nit: `Verifies all the different backup status for scheduled backups`

Reply:
Addressed.